scripts/run_Spectra.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+import json
+from collections import OrderedDict
+import scanpy as sc
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import scipy
+import os
+import anndata
+
+#spectra imports
+import dill
+import Spectra as spc
+from Spectra import Spectra_util as spc_tl
+from Spectra import K_est as kst
+from Spectra import default_gene_sets
+
+def run_Spectra(features_file, metadata_file, gex_datafile, genesets_json, seuratToAdataDir, print_versions = True):
```

Comment:
TODO: probably expose the lambda parameter to runSpectra in R. 

extra TODO: I'll try to better understand what rho + kappa do.